src/test/java/com/google/sps/GeoCodingTest.java

```diff
@@ -0,0 +1,41 @@
+package com.google.sps;
+
+import com.google.sps.data.ApiKeys;
+import com.google.sps.data.GeoCoding;
+import com.google.maps.model.LatLng;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+
+/** */
+@RunWith(JUnit4.class)
+public final class GeoCodingTest {
+  private static final double EPSILON = 0.0001;
+  private static final String RUSSIA_MOSCOW_ADDRESS = "Russia, Moscow, Red Square";
+  private static final LatLng RUSSIA_MOSCOW_LATLNG = new LatLng(55.75393030, 37.62079500);
+  private static final String USA_NEW_YORK_ADDRESS = "Central Park USA New York";
+  private static final LatLng USA_NEW_YORK_LATLNG = new LatLng(40.78121990, -73.96651380);
+  private static final String STRANGE_ADDRESS = "fdskjgl cjtwet ckKERF SEJFN";
+
+  @Test
+  public void testGeoCoding_fromAddressToLatLng_withLocations() {
+    if (!ApiKeys.MAPS_API_KEY.isEmpty()) {
```

Comment:
You can't do actual API calls in a unit test. You could inject a GeoApiContext in the GeoCoding class and mock it for the test.